navigation.md

```diff
@@ -0,0 +1,19 @@
+# Learn Braille navigation
+
+- main menu
+  - theory
+    - continue last visited course
+    - courses [Golubina, ...] `no menu while we have one course`
+      - continue
+      - lessons [lesson 1, ...]
+        - steps
+  - practice
+    - browse symbols
+    - decks [all, ru symbols, punctuation, ...], 
+      - cards
+  - settings 
+    - setting classes `no classes while we have not so much settings`
+      - items
+  - scan QR
+  - help
```

Comment:
I would like to make *help* an item in *settings* and rename *settings* to *settings and help* or something

Reply:
We've already discussed this. And you agreed with me if I remember correctly.

Reply:
you want to make them separate buttons?

I'm not totally against, but there are already too many buttons in main menu for small screens and, besides, there is Help button in a titlebar

Reply:
So why should we have help button in the settings? I think, the purpose of this button is to be easily accessible. This purpose is lost if we place it in settings, so it will be better to remove this button at all.

Reply:
> So why should we have help button in the settings? I think, the purpose of this button is to be easily accessible. This purpose is lost if we place it in settings, so it will be better to remove this button at all.

I agree. We either remove it at all and have help only in the action bar or have it as a separate button, i think.